test/ignores.json

```
@@ -1,6 +1,7 @@
 {
   "query-tests/regressions/mapbox-gl-js#4494": "https://github.com/mapbox/mapbox-gl-js/issues/2716",
   "query-tests/globe/circle/opposite-side-over-north-pole": "skip - needs baseline update",
+  "query-tests/terrain/draped/lines/slope-occlusion": "https://github.com/mapbox/mapbox-gl-js/issues/11726",
```

Nit: Should we consider skipping it to prevent falsely assuming that it's not problematic?
```suggestion
  "query-tests/terrain/draped/lines/slope-occlusion": "skip - https://github.com/mapbox/mapbox-gl-js/issues/11726",
```